pkg/util/log/slog/filewriter/file_writer.go

```diff
@@ -0,0 +1,467 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Copyright (c) 2013 - Cloud Instruments Co., Ltd.
```

Comment:
?

Reply:
Yeah I wasn't sure what was the best way to do that 😅 
Only keeping their copyright might be best, I'll update